pkg/ddevapp/utils.go

```diff
@@ -240,50 +242,60 @@ func CreateGitIgnore(targetDir string, ignores ...string) error {
 			util.Warning("User-managed %s will not be managed/overwritten by ddev", gitIgnoreFilePath)
 			return nil
 		}
-		// Otherwise, remove the existing file to prevent surprising template results
-		err = os.Remove(gitIgnoreFilePath)
-		if err != nil {
-			return err
+		// Read the existing content for future comparison.
+		if gitIgnoreFileBytes, err := os.ReadFile(gitIgnoreFilePath); err == nil {
+			existingContent = string(gitIgnoreFileBytes)
+		} else {
```

Comment:
I suspect this should be "If we can't read or if it's empty then delete it"

There have been a few odd instances of ~/.ddev/commands/ files going empty, and as a result we delete them if they're found empty.

Reply:
> this should be "If we can't read or if it's empty then delete it"

Okay, done.